src/cache/sqlite.ts

```diff
@@ -74,7 +77,17 @@ class SQLiteAdapter {
 
               return responses;
             });
-        }, Promise.resolve());
+        }, Promise.resolve())
+          .then((response) => {
+            return db.close()
```

Comment:
@thomasconner this looks like it will close the `db` on each SQL statement. Is that the intention? Is it necessary?